.env.production

```
@@ -0,0 +1,8 @@
+VITE_STOMP_TEST_MODE=false
+VITE_BACK_URL=https://i13a505.p.ssafy.io
+VITE_MOCK_USER_API=false
+VITE_API_BASE=https://i13a505.p.ssafy.io/api/v1
+# 네이티브 WebSocket 사용 (SockJS 사용 안 함)
+VITE_WS_URL=wss://i13a505.p.ssafy.io/api/ws/chat   # 백엔드가 열어둔 WS 엔드포인트 경로
+VITE_STOMP_TEST_MODE=false
```

dotenv-linter (3.3.0)

[warning] [DuplicatedKey] The VITE_STOMP_TEST_MODE key is duplicated

(DuplicatedKey)